<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAdd</class>
 <widget class="QDialog" name="DataAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Add data</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Fill in new data information</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QComboBox" name="table_comboBox">
      <item>
       <property name="text">
        <string>Income</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Expense</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select table</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="category_comboBox">
      <item>
       <property name="text">
        <string>&lt;New&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Salary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scholarship</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Savings</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDateEdit" name="add_dateEdit">
      <property name="dateTime">
       <datetime>
        <hour>21</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2023</year>
        <month>2</month>
        <day>28</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select category or write a new one</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="category_lineEdit">
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Write a new category</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="value_lineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="addcategory_button">
      <property name="text">
       <string>Add category</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="datacancel_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dataadd_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
